projects/README.md

```diff
@@ -0,0 +1,26 @@
+# GradleUp Projects
+
+The key projects are listed below.
+You can find the full list of projects being developed by the team
+on the [GitHub organization](https://github.com/GradleUp)
+
+## Tools for Android
+
+- [Auto Manifest](../auto-manifest/README.md) -
+  Generates `AndroidManifest.xml` in simple libraries so that you don't have to
+- [Gradle static analysis plugin](../static-analysis-plugin/README.md) -
+  A Gradle plugin to easily apply the same setup of static analysis tools
+  across different Android, Java or Kotlin projects.
+- [Gr8](../gr8/README.md) -
+  Gr8 makes it easy to shadow, shrink, and minimize your jars.
+
+## Gradle DevX
+
+- [Gratatouille](../gratatouille/README.md) -
+  Gratatouille is an opinionated framework to build Gradle plugins. Write pure Kotlin functions and the Gratatouille KSP processor generates tasks, workers and wiring code for you.
+- [NMCP](../nmcp/README.md) (New Maven Central Publishing) -
+  A plugin that uses the new Central Portal publisher API to publish to Maven Central.
+- [Librarian](../librarian/README.md) -
+  Librarian is a library to help building Kotlin libraries.
+- [Maven Sympathy](../maven-sympathy/README.md) -
+  A small Gradle plugin that checks that your project dependencies play nice with Maven resolution strategy
```

Comment:
I'm not sure I get the distinction between Android and Gradle DevX. 

You can use GR8 to shrink/relocate any jar, not only Android for an example. Maybe make a single flat list?

Reply:
For me it seemed to be too big for a single list. But yes, you are right, right now the classification is not very helpful. I will remove it, we can always restore it or do another one in the future